src/components/Icons/Icon.tsx

```diff
@@ -90,6 +90,7 @@ export type IconRotationTypes =
   | "rotate270";
 export type IconSizes =
   | "default"
+  | "xsmall"
```

Comment:
Why is this showing as a new addition in this PR?

Reply:
It's from the `MultiSelect` PR. Remove it?

Reply:
No, it can stay.  The associated update in the `Icon` theme object is also here, so it is a valid addition.  I just could not remember who needed that update.